sdk/cosmosdb/cosmos/CHANGELOG.md

```diff
@@ -778,3 +765,19 @@ Any request to Cosmos DB using a retired SDK will be rejected by the service.
 | [1.0.2](#1.0.2)          | May 23, 2015       | August 30, 2020 |
 | [1.0.1](#1.0.1)          | May 15, 2015       | August 30, 2020 |
 | [1.0.0](#1.0.0)          | April 08, 2015     | August 30, 2020 |
+## 2.0.0-0 
```

Comment:
@chidozieononiwu Any idea why v2 got pulled before v1?

Reply:
We can solve this by adding dates 2.0.0-0 (YYYY-MM-DD). Assuming the there were relased in the correct order them the dates should prevent this sorting error from happening

Reply:
@chidozieononiwu @ramya-rao-a I added the dates for every labeled release and put those 2.0.0-X in the right spot. Hopefully prevents future issues

Reply:
Thanks @zfoster! 

> Hopefully prevents future issues

Hehe, we will know right after you release the next update